pkg/registry/types.go

```diff
@@ -76,6 +76,18 @@ type ChannelEntry struct {
 	Replaces    string
 }
 
+// ChannelEntryNode is a denormalized node in a channel graph annotated with additional entry level info
+type ChannelEntryNode struct {
```

Comment:
I think the naming between `ChannelEntry` and `ChannelEntryNode` is a little confusing. Based on the comments they are similar, one just had more details in it. Maybe we can call it `ChannelEntryAnnotated`?